apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/global-style-sidebar-notice.tsx

```diff
@@ -0,0 +1,65 @@
+declare const wpcomGlobalStyles: { upgradeUrl: string; blogId: string };
+
+import { ExternalLink, Fill, Notice } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { createInterpolateElement, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { recordUpgradeNoticeSidebarShow, recordUpgradeSidebarNoticeClick } from './tracks-events';
+import { useGlobalStylesConfig } from './use-global-styles-config';
+
+const GLOBAL_STYLES_SIDEBAR = 'edit-site/global-styles';
+
+type CoreInterfacePlaceholder = {
+	getActiveComplementaryArea: ( area: string ) => string;
+};
+
+export default function GlobalStylesSidebarNotice(): JSX.Element {
+	const area = useSelect(
+		( select ) =>
+			( select( 'core/interface' ) as CoreInterfacePlaceholder ).getActiveComplementaryArea(
+				'core/edit-site'
+			),
+		[]
+	);
+
+	const isGlobalStylesSidebar = GLOBAL_STYLES_SIDEBAR === area;
+
+	const globalStylesInUse = useGlobalStylesConfig().globalStylesInUse;
+
+	useEffect( () => {
+		if ( globalStylesInUse && isGlobalStylesSidebar ) {
+			recordUpgradeNoticeSidebarShow();
+		}
+	}, [ globalStylesInUse, isGlobalStylesSidebar ] );
+
+	return (
+		<Fill name="ComplementaryArea/core/edit-site">
+			<>
```

Comment:
It'd be nice to add comment explaining why we need this apparently useless fragment 🙂

Reply:
Good point! :)